src/ansys/fluent/core/warnings.py

```diff
@@ -0,0 +1,4 @@
+class PyFluentDeprecationWarning(FutureWarning):
```

Comment:
All PyFluent deprecation warnings should use this class so that all such warnings can be disabled by the following code if user needs it: `warnings.filterwarnings("ignore", category=PyFluentDeprecationWarning)`